### PR TITLE
{dev} intel/2017.01

### PR DESCRIPTION
**Pull request by vanzod**
Simple version update for the Intel toolchain to the latest available update.

### CODE REVIEW

**Comment by pescobar** [comment]
@boegel I plan to install this toolchain in my system. Is it fine like this or do you plan to do any change before merging it?

**Comment by akesandgren** [comment]
Not sure it's 100% correct, I'm trying to build QM-E with it (well my version of it that looks identical to me) and QM-E fails to find mkl_scalapack...

File: easybuild/easyconfigs/i/icc/icc-2017.1.132-GCC-5.4.0-2.26.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+]
+
+# list of regex for components to install
+# full list of components can be obtained from pset/mediaconfig.xml in unpacked sources
+# cfr. https://software.intel.com/en-us/articles/intel-composer-xe-2015-silent-installation-guide
+components = ['intel-comp', 'intel-ccomp', 'intel-icc', 'intel-openmp', 'intel-ipsc?_']
+
+dontcreateinstalldir = 'True'
+
+license_file = '/opt/easybuild/licenses/intel/license.lic'
```
Comment (boegel) [inline easybuild/easyconfigs/i/icc/icc-2017.1.132-GCC-5.4.0-2.26.eb]: @vanzod this shouldn't be hardcoded like this, please use (same in other easyconfigs)

```python
license_file = HOME + '/licenses/intel/license.lic'
```

**Comment by boegel** [comment]
@vanzod do you mind including an easyconfig for HPL 2.2 that uses this new toolchain as well, which can serve as a test case?

File: easybuild/easyconfigs/i/icc/icc-2017.1.132-GCC-5.4.0-2.26.eb (hunk 1 above)
Comment (vanzod) [inline easybuild/easyconfigs/i/icc/icc-2017.1.132-GCC-5.4.0-2.26.eb]: Done

**Comment by vanzod** [comment]
@boegel I am also updating the easyconfigs for some of the other software from the Cluster Edition suite. Do you want me to push everything here or do you prefer a separate PR?

**Comment by boegel** [comment]
@vanzod please pack them together in a separate PR

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node1001.muk.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/54433a9304c45337950cbfe9f635ba9b for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2301.phanpy.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/9a3d0f60a91b3a6731ab22b2327cb824 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**